sccm/core/support/support-center-onetrace.md

```diff
@@ -44,7 +44,7 @@ OneTrace works with many types of log files, such as:
 
 OneTrace installs with Support Center. Find the Support Center installer on the site server at the following path: `cd.latest\SMSSETUP\Tools\SupportCenter\SupportCenterInstaller.msi`.
 
-By default, the OneTrace application is installed at `"C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe`.
+By default, the OneTrace application is installed at `C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe"`.
```

Comment:
It looks like the leading quotation mark was removed, but one was added at the end of the string.

```suggestion
By default, the OneTrace application is installed at `C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe`.
```

You can either simply accept my suggestion, or in the **Files changed** section of the pull request, select the ellipsis (`...`) in the upper right, and select **Edit file**. Make the change into the *same branch* to automatically update this pull request.